libs/dh/charges/feature-prices/src/lib/search-result/dh-charges-prices-result.component.html

```diff
@@ -99,34 +103,38 @@ <h3>
 
```

Comment:
No sorting by `taxIndicator` or `transparentInvoicing`? - I'm not sure it is needed, just asking :-)

Reply:
For now it is not needed :)